cli.js

```diff
@@ -21,6 +21,7 @@ if (args.help || args._.length === 0) {
   console.log("--no-default-entries  Won't parse your main and bin entries from package.json will be parsed")
   console.log('--version             Show current version')
   console.log('--ignore              To always exit with code 0 pass --ignore')
+  console.log('--transformer=<cmd>   Transform each file with the program "cmd". The current filename is available as $$')
```

Comment:
It would be nice to add an example to the readme to illustrate how `$$` works in practice